pocket/chain.json

```
@@ -0,0 +1,112 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "pocket",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.pokt.network/",
+  "pretty_name": "Pocket Network MainNet",
```

```suggestion
  "pretty_name": "Pocket Network",
```

I think we should drop MainNet as this is the default network. Checked other non-testnet files in the repo - seems like this is the practice.